extension/controller/backgroundScript.js

```diff
@@ -61,6 +65,15 @@ let telemetryByTab = new Map();
 let translationRequestsByTab = new Map();
 let outboundRequestsByTab = new Map();
 const translateAsBrowseMap = new Map();
+let isMochitest = false;
+
+// set data collection accordingly with what the user set
+browser.storage.local.get({ telemetryCollectionConsent: true,
+  errorCollectionConsent: true, }).then(items => {
+    pingSender.setUploadEnabled(items.telemetryCollectionConsent);
+    Sentry.getCurrentHub().getClient()
```

Comment:
hm, we're already doing this in window.load after sentry init... not sure what will happen if it's not initialized yet